### PR TITLE
[JsonPath transform] Improve/clarify README

### PR DESCRIPTION
**Pull request by 9037568**
An attempt at improving/clarifying the README for the JsonPath transformation.

Signed-off-by: 9037568 <namraccr@gmail.com>

### CODE REVIEW

**Comment by mhilbush** [comment]
Would it also make sense to mention that the JsonPath transform returns the entire original json string when there's no match for the expression? This is a common question asked on the forums concerning the use of JsonPath in rules.

**Comment by 9037568** [comment]
>Would it also make sense to mention that the JsonPath transform returns the entire original json string when there's no match for the expression?

Yes, that's a good call. I'll add it.

File: bundles/org.openhab.transform.jsonpath/README.md [hunk 1]
```diff
@@ -48,31 +35,34 @@ rule "Convert JSON to Item Type Number"
...
 
-Compared to standard JSON the transformation it returns evaluated values when a single alement is retrieved from the query.
-Means it does not return a valid JSON `[ 23.2 ]` but `23.2`, `[ "Outside" ]` but `Outside`.
-This makes it possible to use it in labels or output channel of things and get Numbers or Strings instead of JSON arrays.
-A query which returns multiple elements as list is not supported.
+Compared to standard JsonPath, the transformation returns single values instead of arrays.
+The transformation also will not ever return `null`.
+This makes it possible to use the transform in labels or output channels of Things.
+
+Because the transformation will not return arrays or null values, if the JsonPath expression provided results in no matches, the transformation will return the entire original JSON string.
```
Comment (J-N-K) [inline bundles/org.openhab.transform.jsonpath/README.md]:
```suggestion
If the JsonPath expression does not match (or the match is invalid), the entire original JSON string is returned.
```

**Comment by J-N-K** [comment]
@9037568 Can you apply the suggested change or something similar? Thanks.

**Comment by 9037568** [comment]
@J-N-K Done.

**Comment by Clarkkent111-droid** [comment]
Thank you for the Experience, hopefully obtaining employment for a business such as these would be a blessing.

**Pull request merged**